R/epidatacall.R

```diff
@@ -198,6 +198,14 @@ fetch_args_list <- function(
   )
 }
 
+#' @export
+print.fetch_args <- function(x, ...) {
+  stopifnot(inherits(x, "fetch_args"))
```

Comment:
musing: I think the conventional S3 method pattern is omit this type of check, and assume that you did / could have gotten here from S3 dispatch from the generic.  Same thing probably applies to other `print` implementations added below.